math/intlog2.go

```diff
@@ -0,0 +1,10 @@
+package math
+
+import "math/bits"
+
+func Log2(u uint64) int {
```

Comment:
I'd add a comment noting that this rounds down

Reply:
Done.